tests/python/tir-transform/test_tir_transform_merge_static_shared_memory_allocations.py

```diff
@@ -0,0 +1,203 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import numpy as np
+
+import tvm
+import tvm.testing
+from tvm import te
+from tvm.driver.build_module import schedule_to_module
+from tvm.topi.math import cast
+from tvm.script import tir as T
+
+
+def run_passes(sch, args):
+    mod = schedule_to_module(sch, args)
+    with tvm.transform.PassContext(config={"tir.merge_static_smem": True}):
```

Comment:
tir.merge_static_smem is set to False by default to ensure the code is more readable. (for example, maintain clearly definition of A_shared, B_shared, instead of (half*)(buf_shmem+ offset)) , so it should be manually enabled.

Reply:
Got it - so it's turned off basically for better readability, is my understanding correct?